workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/ResourceFolderProvider.java

```diff
@@ -0,0 +1,5 @@
+package com.symphony.bdk.workflow.engine;
+
+public interface ResourceFolderProvider {
```

Comment:
we haven't really defined the other ResourceProvider as a functional interface, so I would probably at a method to it instead